python/tests/sklearn/base_tests.py

```diff
@@ -155,6 +174,60 @@ def test_logistic_regression_deserializer(self):
         self.assertEqual(res_a[1], res_b[1])
         self.assertEqual(res_a[2], res_b[2])
 
+    def test_multinomial_logistic_regression_serializer(self):
+
+        logistic_regression = LogisticRegression(fit_intercept=True)
+        logistic_regression.mlinit(
+            input_features='a',
+            prediction_column='prediction'
+        )
+
+        X = self.df[['a']]
+        y = np.array([to_standard_normal_quartile(elem) for elem in X.to_numpy()])
+
+        logistic_regression.fit(X, y)
+        logistic_regression.serialize_to_bundle(self.tmp_dir, logistic_regression.name)
+
+        with open("{}/{}.node/model.json".format(self.tmp_dir, logistic_regression.name)) as json_data:
+            model = json.load(json_data)
+
+        self.assertEqual(model['op'], 'logistic_regression')
+        self.assertEqual(model['attributes']['num_classes']['long'], 4)
+
+        # assert 4x1 coefficient matrix
+        self.assertEqual(len(model['attributes']['coefficient_matrix']['double']), 4)
+        self.assertEqual(len(model['attributes']['coefficient_matrix']['shape']['dimensions']), 2)
+        self.assertEqual(model['attributes']['coefficient_matrix']['shape']['dimensions'][0]['size'], 4)
+        self.assertEqual(model['attributes']['coefficient_matrix']['shape']['dimensions'][1]['size'], 1)
+
+        # assert 4x0 intercept vector
+        self.assertEqual(len(model['attributes']['intercept_vector']['double']), 4)
+        self.assertEqual(len(model['attributes']['intercept_vector']['shape']['dimensions']), 1)
+        self.assertEqual(model['attributes']['intercept_vector']['shape']['dimensions'][0]['size'], 4)
+
+    def test_multinomial_logistic_regression_deserializer(self):
+
+        logistic_regression = LogisticRegression(fit_intercept=True)
+        logistic_regression.mlinit(
+            input_features='features',
```

Comment:
this one too? Sorry, should have pointed it out earlier

Reply:
All good, just noticed it myself